src/components/Instructions/Instructions.css

```diff
@@ -1,9 +1,11 @@
 .instructions {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: space-evenly;
-  height: 100%;
-  width: 90%;
-  font-size: 1vw;
```

Comment:
Generally speaking, percentages and vw/vh aren't great for styling fonts. They don't need to grow and shrink in the same way as some other elements. I Changed it where I saw it. I made a lot of fonts smaller for the time being.